frontend/src/components/GrantAccess.vue

```diff
@@ -36,60 +52,59 @@ async function grantAccess() {
   const grantAccessRequest = toRequest(formData.value)
   try {
     await apiService.grantUserRole(grantAccessRequest)
-    toast.success(`User "${grantAccessRequest.user_name}"" is granted with "${formData.value.role.role_name}" access.`)
-    formData.value = JSON.parse(JSON.stringify(defaultFormData)) // clone default input data.
+    toast.success(`User "${grantAccessRequest.user_name}" is granted with "${formData.value.role.role_name}" access.`)
+    formData.value = {...defaultFormData}; // clone default input data.
+    // TODO: need to find a way to clear validation
```

Comment:
Has this todo been addressed? Maybe it isn't relevant anymore. Either way, this should be removed.

Reply:
I have removed this TODO as it seems it isn't needed anymore.